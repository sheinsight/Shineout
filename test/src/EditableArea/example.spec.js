import exampleTest from '../../example'

exampleTest('EditableArea')
