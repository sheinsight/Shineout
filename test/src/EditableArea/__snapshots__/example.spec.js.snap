// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test: EditableArea[site/pages/components/EditableArea/example-01-base.js] 1`] = `
<div>
  <pre>
    <span>
      input something
    </span>
  </pre>
  <textarea />
  
</div>
`;

exports[`Snapshot test: EditableArea[site/pages/components/EditableArea/example-02-controlled.js] 1`] = `
<div>
  <pre>
    <span>
      Input something
    </span>
  </pre>
  <textarea />
  
</div>
`;
